dbt/adapters/athena/column.py

```diff
@@ -0,0 +1,20 @@
+from dbt.adapters.base import Column as BaseColumn
+
+
+class AthenaColumn(BaseColumn):
```

Comment:
Did you consider to use a macro instead?

Reply:
Yes, It's ok to move implementation of this class to macro.
should I wait for #26 merged?

Reply:
I think that the #26 is slightly different. You might want to build something like that: https://github.com/dbt-athena/dbt-athena/blob/main/dbt/include/athena/macros/materializations/models/helpers_iceberg.sql#L102-L124 
so feel free to proceed.

Reply:
`helpers_iceberg.sql ` was very helpfull, I move the AthenaColumn implementation to  macro

[46c0121](https://github.com/dbt-athena/dbt-athena/pull/30/commits/46c012158cdcfcdaa724ac4b5e636d5808d803d6)